### PR TITLE
Adding support for option enableListInterface in PARQUET mode while writing to BigQuery

### PR DESCRIPTION
**Pull request by suryasoma**

### CODE REVIEW

File: README-template.md [hunk 1]
```diff
@@ -678,6 +678,14 @@ The API Supports a number of options to configure the read
      </td>
      <td>Read</td>
    </tr>
+   <tr valign="top">
+     <td><code>enableListInference</code>
+     </td>
+     <td>  Indicates whether to use schema inference specifically for Parquet LIST logical type.
```
Comment (VinayLondhe14) [inline README-template.md:684]: Do you want to add a link here to the Pantheon doc?

Comment (suryasoma) [reply README-template.md:684]: Sure

**Review by VinayLondhe14 — APPROVED** [review]

**Pull request merged**